go.mod

```diff
@@ -32,34 +32,40 @@ require (
 	golang.org/x/mod v0.6.0-dev.0.20220106191415-9b9b3d81d5e3
 	golang.org/x/tools v0.1.10
 	gomodules.xyz/jsonpatch/v3 v3.0.1
-	helm.sh/helm/v3 v3.6.2
+	helm.sh/helm/v3 v3.8.2
 	k8s.io/api v0.23.5
 	k8s.io/apiextensions-apiserver v0.23.5
 	k8s.io/apimachinery v0.23.5
-	k8s.io/cli-runtime v0.23.1
+	k8s.io/cli-runtime v0.23.5
 	k8s.io/client-go v0.23.5
-	k8s.io/kubectl v0.23.1
+	k8s.io/kubectl v0.23.5
 	sigs.k8s.io/controller-runtime v0.11.2
 	sigs.k8s.io/controller-tools v0.8.0
 	sigs.k8s.io/kubebuilder/v3 v3.4.1
 	sigs.k8s.io/yaml v1.3.0
 )
 
+replace (
+	k8s.io/api v0.24.0 => k8s.io/api v0.23.5
```

Comment:
We would soon be bumping deps in SDK to 1.24 K8s. Could we wait for this, till https://github.com/operator-framework/operator-sdk/pull/5843 is merged.

Reply:
#5843 is also going to be bumping Helm to 3.9.0 which will end up making this PR obsolete.